sql-ballerina/tests/simple-params-query-test.bal

```diff
@@ -519,21 +519,21 @@ function queryDateLongParam() {
     groups: ["query", "query-simple-params"]
 }
 function queryDateTimeRecordParam() {
-    time:Time date = checkpanic time:parse("2017-02-03", "yyyy-MM-dd");
+    time:Date date = {year: 2017, month:2, day: 3};
     DateValue typeVal = new (date);
     ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE date_type = ${typeVal}`;
     validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
 }
 
-@test:Config {
-    groups: ["query", "query-simple-params"]
-}
-function queryDateTimeRecordWithTimeZoneParam() {
-    time:Time date = checkpanic time:parse("2017-02-03T09:46:22.444-0500", "yyyy-MM-dd'T'HH:mm:ss.SSSZ");
-    DateValue typeVal = new (date);
-    ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE date_type = ${typeVal}`;
-    validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
-}
+//@test:Config {
```

Comment:
Shall we use `enable = false` if we are disabling the test?

Reply:
@MadhukaHarith92 these commented test cases can not only disabled because it is used invalid Time module APIs. In Future, we will enable these test cases with proper Time module APIs.